src/components/UserProfile/UserProfileSettings/index.js

```diff
@@ -83,9 +83,7 @@ class UserProfileSettings extends Component {
                 sent to your email.
               </p>
             </div>
-
-            <ErrorMsgBox errorMsg={error} />
-
+            <MessageBox errorMsg={error} />
```

Comment:
Do we need to pass in `type`?

Reply:
type has a default value in Message.defaultProps